ui/lib/core/addon/components/overview-card.hbs

```diff
@@ -6,22 +6,21 @@
 <Hds::Card::Container
   @level="mid"
   @hasBorder={{true}}
-  class="overview-card border-radius-2"
+  class="overview-card"
   data-test-overview-card-container={{@cardTitle}}
   ...attributes
 >
-  <div class="is-flex-between" data-test-overview-card={{@cardTitle}}>
-    <h3 class="title is-5">{{@cardTitle}}</h3>
+  <div class="flex row-wrap space-between has-bottom-margin-m" data-test-overview-card={{@cardTitle}}>
+    <h3 class="has-text-weight-bold is-size-5">{{@cardTitle}}</h3>
     {{#if @actionText}}
-      <LinkTo
-        class="has-icon-right is-ghost is-no-underline has-text-weight-semibold"
+      <Hds::Link::Standalone
```

Comment:
updated to HDS link which fails gracefully if there is no `query` param. Otherwise, `<LinkTo>` errors when `@actionQuery` is undefined because it demands an object to be passed